api/src/main/java/org/openmrs/util/Security.java

```diff
@@ -42,7 +42,7 @@
 	/**
 	 * encryption settings
 	 */
-	public static Logger log = LoggerFactory.getLogger(Security.class);
+	public static final Logger log = LoggerFactory.getLogger(Security.class);
```

Comment:
I think we changed to not having loggers being static, can we clean it up here as well

Reply:
@wluyima  Ok, When I modify it as public Logger log, it produces errors in other parts of the code with "non static fields can not be referenced in static methods".. Shall I go on and fix these errors as well ?